recipes/libntlm/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo $PREFIX
```

Comment:
You don't need to echo this, it will be visible in the next command:
```
+ echo /opt/conda/envs/_build
+ ./configure --prefix=/opt/conda/envs/_build
```